src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/helper/ModelFactory.kt

```diff
@@ -334,7 +334,6 @@ fun requestProbationVideoLinkRequest(
 }
 
 fun amendCourtBookingRequest(
-  courtCode: String = "DRBYMC",
```

Comment:
This was a redundant/unused test factory param, you cannot amend a court once a booking is created.